picasso/src/main/java/com/squareup/picasso3/Picasso.java

```diff
@@ -660,11 +667,30 @@ void cancelExistingRequest(Object target) {
 
     private boolean indicatorsEnabled;
     private boolean loggingEnabled;
+    private boolean isChild;
 
     /** Start building a new {@link Picasso} instance. */
     public Builder(@NonNull Context context) {
       checkNotNull(context, "context == null");
       this.context = context.getApplicationContext();
+
+      isChild = false;
+    }
+
+    Builder(Picasso picasso) {
+      context = picasso.context;
+      callFactory = picasso.callFactory;
+      service = picasso.dispatcher.service;
+      cache = picasso.cache;
+      listener = picasso.listener;
+      requestTransformers.addAll(picasso.requestTransformers);
+      requestHandlers.addAll(picasso.requestHandlers);
```

Comment:
won't this end up adding all the default request handlers again?

Reply:
yep.  good catch.

Reply:
updated.